styleguide/source/assets/js/sign-in-dropdown.js

```diff
@@ -8,10 +8,9 @@
 
       function dropdownDownMenu(parentElement, menuElement) {
 
-        parentElement.click(function(e){
+        parentElement.unbind('click').click(function(e){
```

Comment:
Drupal forces us to unbind a click event before binding a new click event. Convoluted but works